src/Agent/NewRelic/Agent/Extensions/NewRelic.Agent.Extensions/Api/IAgentApi.cs

```diff
@@ -1,6 +1,8 @@
 // Copyright 2020 New Relic, Inc. All rights reserved.
 // SPDX-License-Identifier: Apache-2.0
 
+#nullable enable
```

Comment:
Would it be a lot more effort to enable this for the entire NewRelic.Agent.Extensions project?

Reply:
Alternatively, is it possible to just not touch NewRelic.Agent.Extensions for now? We do not currently publish it to nuget.

Reply:
It is a lot more work to update the entire project.  It involves the `VisibilityBypasser` and Delegate functions.  I started down the path, and almost every file needed changes.

I can look into further, but not doing it for this project will cause warnings to appear where those items are used.  They'll be treated as Not Nullable for any caller.  If the caller is `<Nullable>enabled</Nullable>`, this will show warnings.